frontends/web/src/assets/exchanges/exchanges.json

```diff
@@ -0,0 +1,139 @@
+[{
+		"key": "exmo",
+		"link": "https://exmo.com",
+		"description": "Founded in 2013 and based in London, Kiev, Barcelona, and Moscow. EXMO is fairly popular in Eastern Europe.",
+		"regions": ["EU", "NA"],
+		"payment": ["CC", "BT"]
+	},
+	{
+		"key": "coinmama",
+		"link": "https://www.coinmama.com/",
+		"description": "Coinmama, founded in 2013, is a financial service company that makes it fast, safe and fun to buy digital currency, anywhere in the world.",
+		"regions": ["EU", "NA", "APAC"],
+		"payment": ["CC", "BT"]
+	},
+	{
+		"key": "luno",
+		"link": "https://www.luno.com/en/",
+		"description": "Luno makes it safe and easy to buy, store and learn about cryptocurrencies",
+		"regions": ["EU", "APAC"],
+		"payment": ["CC", "BT"]
+	},
+	{
+		"key": "kraken",
+		"link": "https://www.kraken.com/",
+		"description": "As one of the largest and oldest Bitcoin exchanges in the world, Kraken is consistently named one of the best places to buy and sell crypto online.",
+		"regions": ["EU", "NA"],
+		"payment": ["BT"]
+	},
+	{
+		"key": "bity",
+		"link": "https://bity.com/",
+		"description": "Bity, founded in 2014 and incorporated in Switzerland is a leading provider of crypto-finance services.",
+		"regions": ["EU"],
+		"payment": ["BT"]
+	},
+	{
+		"key": "bitstamp",
+		"link": "https://www.bitstamp.net/",
+		"description": "Bitstamp is one of the oldest  European based cryptocurrency exchange. It allows people from all around the world to buy and sell cryptocurrencies safely.",
+		"regions": ["EU", "NA", "LA", "APAC", "AF"],
+		"payment": ["CC", "BT"]
+	},
+	{
+		"key": "bitcoin_suisse",
+		"link": "https://www.bitcoinsuisse.com/",
+		"description": "Founded in 2013, Bitcoin Suisse AG has been a pioneer in providing crypto-financial services.",
+		"regions": ["EU"],
+		"payment": ["BT"]
+	},
+	{
+		"key": "shapeshift",
+		"link": "https://shapeshift.io/",
+		"description": "",
+		"regions": ["EU", "NA", "LA", "APAC", "AF"],
+		"payment": ["SW"]
+	},
+	{
+		"key": "kyberswap",
+		"link": "https://kyberswap.com/",
+		"description": "KyberSwapis a fast, simple, and secure token swap platform.",
+		"regions": ["EU", "NA", "LA", "APAC", "AF"],
+		"payment": ["SW"]
+	},
+	{
+		"key": "hodlhodl",
+		"link": "http://hodlhodl.com",
```

Comment:
Probably best as https://

Reply:
https is better b5662495bb54da38632e5018b01f51fbe2c59efe